pandas/core/indexes/multi.py

```diff
@@ -1036,7 +1037,14 @@ def _get_level_values(self, level, unique=False):
         labels = self.labels[level]
         if unique:
             labels = algos.unique(labels)
-        filled = algos.take_1d(values._values, labels,
+
+        if is_period_dtype(values):
+            take_vals = values._ndarray_values
+        else:
+            # TODO: Why is this _values where elsewhere it is values?
```

Comment:
I'm probably responsible for using `values._values` here instead of `.values`. AFAIK, they should be equivalent for a MultiIndex (an ndarray of tuples).

Reply:
OK.  I'll clear this out in the next pass, assuming this goes through.